mendix/widgets-resources
# [N] Line chart - Implement Atlas 3 design

### Why?
Line chart will be a new widget that needs to be shipped with Atlas 3 styles for MX 9.

### What has been done?
- Introduce `lineColorPalette` style property used as a fallback when lines have no color defined.
- Change line draw order with the first object being overriden by following configured lines in the object property list.
- Rename configuration properties:
  - `Series` to `Lines`
  - `StylePropertyName` property to `Custom line style`
- Add descriptions to the configuration properties `Lines` and `Custom line style`.
- Rename `series` style property to `lineStyles`.
- Improve render checks on axis label values.
- Adjust default style (style without Atlas).
- Make error message stylable.
- Simplify widget styles by exposing specific styles of Victory JS library in customary native style terms.
- Add Atlas 3 styles by using as many Atlas variables as possible for colors and font styling.
- Support dark mode.

### How to test?
- Test configuration changes in Studio Pro.
- Run a Mendix app with the new Atlas styles and the widget and verify the styling to the design. Don't forget to test dark mode.
- Add custom styling to test the added and adjusted style properties.
